src/main/java/dao/PieceDao.java

```diff
@@ -0,0 +1,147 @@
+package dao;
+
+import domain.board.Board;
+import domain.board.BoardLocation;
+import domain.piece.Cannon;
+import domain.piece.Chariot;
+import domain.piece.Elephant;
+import domain.piece.Horse;
+import domain.piece.King;
+import domain.piece.Pawn;
+import domain.piece.Piece;
+import domain.piece.Scholar;
+import domain.piece.Score;
+import domain.piece.Team;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+
+public class PieceDao {
+
+    private final JdbcConnection jdbcConnection;
+
+    public PieceDao(JdbcConnection jdbcConnection) {
+        this.jdbcConnection = jdbcConnection;
+    }
+
+    public void initializePieceIfNotExists(Board board) {
+        Map<BoardLocation, Piece> pieces = board.getPieces();
+        String query = "INSERT INTO piece ("
+                + "piece_type,"
+                + "team,"
+                + "location_x,"
+                + "location_y) " +
+                "VALUES (?, ?, ?, ?)";
```

Comment:
java 특정 버전 이후 부터 muti line String을 지원하는 것으로 알고 있어서요! 좀 더 가독성을 챙길 수 있지 않을까 하여 추천해봐요 ㅎㅎ

```suggestion
        String query = """
                INSERT INTO piece (piece_type, team, location_x, location_y) 
                VALUES (?, ?, ?, ?)
                """;
```

* [java-multiline-string](https://www.baeldung.com/java-multiline-string)